contracts/src/v0.1/mocks/RequestResponseConsumerMock.sol

```diff
@@ -58,7 +62,33 @@ contract RequestResponseConsumerMock is RequestResponseConsumerBase {
         requestId = COORDINATOR.requestData{value: msg.value}(req, callbackGasLimit);
     }
 
-    function fulfillDataRequest(uint256 /*requestId*/, uint256 response) internal override {
+    function fulfillDataRequestUint256(uint256 /*requestId*/, uint256 response) internal override {
         s_response = response;
     }
+
+    function fulfillDataRequestInt256(uint256 /*requestId*/, int256 response) internal override {
```

Comment:
I am curious. Have you tried to remove some of these functions? Can it still compile?

Reply:
yes, I did, and it can not compile. All virtual functions need to be overrides

Reply:
In that case we have to figure out how to deal with it. We should not force people to override every possible `fulfill*` function when they most likely need just one. Can you please propose some solution?

Reply:
we can use an interface instead, but they have to import the interface

Reply:
I think importing interface is okay! 🚀